src/frontend-samples/missclassification-sample/MisclassificationTableWidget.tsx

```diff
@@ -0,0 +1,171 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import React, { useCallback, useEffect } from "react";
+import { Spinner, SpinnerSize } from "@bentley/ui-core";
+import {
+  AbstractWidgetProps,
+  PropertyDescription,
+  PropertyRecord,
+  PropertyValue,
+  PropertyValueFormat,
+  StagePanelLocation,
+  StagePanelSection,
+  UiItemsProvider,
+  WidgetState,
+} from "@bentley/ui-abstract";
+import {
+  ColumnDescription,
+  RowItem,
+  SimpleTableDataProvider,
+  Table,
+} from "@bentley/ui-components";
+import { IModelApp } from "@bentley/imodeljs-frontend";
+import MisclassificationApi from "./MisclassificationApi";
+import { MisclassificationsJson } from "./MisclassificationsJson";
+
+const MisclassificationTableWidget: React.FunctionComponent = () => {
+  const [misclassData, setMisclassData] = React.useState<any>();
+
+  useEffect(() => {
+    async function getResults() {
+      const testResults = await MisclassificationApi.getTestResults(
+        "e9686c52-1364-4fd4-8b50-1291f82a5be1",
+        "cf226a09-f06e-4c24-8ea5-5ad71a1d8742",
+        "e70567a0-a917-408d-9c6c-e5e2fa247712"
+      );
+
+      if (testResults) {
+        // download the results
+        const sasUri = testResults.resultSasUri;
+
+        if (sasUri.includes("resanalysis")) {
+          setMisclassData(MisclassificationsJson);
+          return;
+        }
+
+        const testData = await MisclassificationApi.getTestResultsData(sasUri);
+        if (testData !== undefined) {
+          const testDataObj = JSON.parse(testData);
+          setMisclassData(testDataObj);
+        } else {
+          // use the hardcoded json values
```

Comment:
Add note here. Something like "in production environment this would be a problem. But in this example, we expect the SAStoken to fail, so we fall back to an example of hardcoded JSON"